app/src/main/java/org/kiwix/kiwixmobile/di/components/KiwixActivityComponent.kt

```diff
@@ -25,7 +25,7 @@ import org.kiwix.kiwixmobile.di.modules.KiwixActivityModule
 import org.kiwix.kiwixmobile.intro.IntroActivity
 import org.kiwix.kiwixmobile.intro.IntroModule
 import org.kiwix.kiwixmobile.language.LanguageActivity
-import org.kiwix.kiwixmobile.local_file_transfer.LocalFileTransferActivity
+import org.kiwix.kiwixmobile.localfiletransfer.LocalFileTransferActivity
```

Comment:
camelCase is more appropriate than all lowercase for package naming, so says the [conventions](https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules). Pretty sure I prefer camelCase to concatenating.names.with.periods